src/DependencyInjection/Configuration.php

```diff
@@ -125,9 +125,19 @@ private function getOptionsConfig(): NodeDefinition
         $rootNode
             ->addDefaultsIfNotSet()
             ->children()
+                ->enumNode('layout')
+                    ->values(['horizontal', 'vertical'])
+                    ->defaultValue('horizontal')
```

Comment:
> @kevinpapst please remove that, otherwise tabler.options node is required to be configured

What is wrong with that ?

Reply:
I already removed it. It is "wrong" because:
- it has a default values
- addDefaultsIfNotSet() is set
- it was the ONLY config that had this setting, thus forcing the user to create the config/packages/tabler.yaml

The demo app works fine without the config file if `isRequired()` is removed.
If we ship good default values, user can configure the bundle, but don't have to at the beginning. 
This should make the entry barrier for new devs smaller.